docs/releases.rst

```diff
@@ -4,6 +4,15 @@
     Releases
 ======================
 
+tmt-1.53.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`organize-data` chapter of the Guide has been extended
+with the :ref:`/spec/core/adjust` section describing how to
+conditionally modify various attributes of tests, plans,
+or stories based on the current :ref:`/spec/context`.
```

Comment:
I don't think we need release notes for documentation changes.

Reply:
Documentation is definitely a user-facing content so it makes a perfect sense to me.